src/parser/core/modules/Defensives.tsx

```diff
@@ -53,11 +58,22 @@ export class Defensives extends Analyser {
 	}
 
 	private getUsageCount(defensive: Action): number {
-		return this.getUses(defensive).length
+		// The cooldowns module actually returns events based on cooldown *group* so make sure we're actually getting the uses for the cooldown we asked for
+		return this.getUses(defensive).filter(event => event.action.id === defensive.id).length
```

Comment:
is it safe for `getUses().length != getUsageCount()`? semantics seem a bit askew that "usage" means two different things here

Reply:
Eh, probably not. BLU's override is using getUses().length since getUsageCount didn't exist yet so I should probably make sure those match now. Made a getGroupUses that functions like the original getUses, put the filter to the specific action in getUses, and changes this module's calls to getUses over to getGroupUses where they formerly expected to work off the full group.